BoardGame-CLI/uno.py

```diff
@@ -99,9 +100,10 @@ def canPlay(colour, value, playerHand):
 numPlayers = int(input("How many players?"))
 while numPlayers < 2 or numPlayers > 4:
     numPlayers = int(
-        input("Invalid. Please enter a number between 2-4.\nHow many players?"))
+        input("Invalid. Please enter a number between 2-4.\nHow many players?")
+    )
 for player in range(numPlayers):
-    players_name.append(input("Enter player {} name: ".format(player+1)))
+    players_name.append(input("Enter player {} name: ".format(player + 1)))
```

Comment:
f-string is easier to read in this case

Reply:
Okay. Readability ❤️ is the 🔑

Reply:
F-string is 

'''python 
f"hello, {user}"
'''

No need to use `.format()`